### DISCUSSION TITLE
Three Phase

### DISCUSSION THREAD

**Discussion by 34by151** [jmcollin78/solar_optimizer#58] [Q&A] [answered]
Been changing over from pv_excess
So far I really like solar_optimizer

**This is how my power is billed**
I have 3 Phase Power 
I am billed via net metering per phase not netmetering with all phases combined
My solar is on phase 1, 2 and 3

So while I may be exporting more than I consume I may still get a bill.
IE if Phase 1 and 2 are exporting a combined 1kW of power and Phase 3 is importing 1 kW of power
I am billed for 1kW of use not 0kW

When you combine this will billing it gets worse
I pay 36 cents per kwh for imports but get paid 5c for exports
In other words in the above example I would be billed (assuming 1kw for 1 hour)
(36 cents * 1 hour) - (5 cents * 1 hour) = 31 cents billed
If tit was combined net metering it would be
(5 cents * 1 hour) = 5 cents credit

Its not in my intrest to export power as the payment is so low
Its worse to import
Hence load control to maximise self consumption during the day and minimise consumption at night

**My question is**
Can I setup as 3 devices?
Ie one per phase, each one having its own loads
Power Net Power consumption and solar per phase

Im presently setup and testing as a combined 3 phase allthough I have sensors for each phase (net power and solar)

**Comment by jmcollin78** [comment]
Hello,

I have also 3 Phase bill with exactly the same behavior and I have already think about this case.

> Can I setup as 3 devices?

It won't help in your case. What we need, it somehow, to get 3 SolarOptimizer: one per phase.
If you get the import / export and consumption by phase of your devices, this is the only way I see to achieve the best optimisation.

But for now, it is not possible to have more than one SO installed and I guess it is an "overkill" configuration and installation to save cents. That is only my opinion.

**Reply by 34by151** [reply]
Its a massive difference in the bill if the solar is only on one phase

I actually have 3 phase + off peak
Off Peak is a seperate meter but uses phase 1 (just seperatly billed)

Solar is only in Phase 1
The loads that are alllowed to use Off Peak such as the pool are billed at 22c
Imports are 36c
Exports are 5c

So if I need to run the pool for 3 hours today
I want to get as much time as I can self consuming solar which costs 0
Any time I dont get I have to run on off peak which costs 22c rather
This alone saves me $500 per bill

Same goes fot the hot water
Switching saves around $500 per bill

I have to lump the imports on phase 2 and 3 till I can get 3 phase solar
I cant do much till then for the 3 phase loads like my welder
But for single phase loads I can phase shift them to phase 1, which im doing

**Reply by jmcollin78** [reply]
You have a 3 phase electric installation and a one phase Solar equipment ?
Oh yes, this is a bad case, and it can make a difference depending on import cost.

**Reply by jmcollin78** [reply, marked as answer]
The solution is to have 3 SO installation and configuration which is not possible yet as I said before because of the configuration in configuration.yaml. Once the configuration will be through GUI config flow (see https://github.com/jmcollin78/solar_optimizer/issues/54) it should be possible. But #54 is a big change

**Reply by 34by151** [reply]
Yes not optimal 

Simple maths are bill is around $1500 without any control
Pool is $500
Hot Water is $500
The rest is $500

Phase Shifting Pool between Off peak and solar pahse brings thet down to around $100
Same goes for the Hot Water
Load control on the rest brings that down to $200

So $1500 to $400
If I can get rid of the remaining solar exports that will go down to around $250

Yes more solar will do that and more as well
I have plans for 3 Phase solar and 3 phase battery storage
The issue is the investment vs the recovery cost
Same goes to a lesser exten for solar on the rmaing 2 phases

**Reply by 34by151** [reply]
I think I have a work around by combining Optimiser and PV Excess Optimizer
